src/docs/blueprint_documentation.md

```diff
@@ -1,13 +1,17 @@
 ---
-title: Packaging Theia as a Desktop Product
+title: Extending/Adopting the Theia IDE
 ---
 
-# Packaging Theia as a Desktop Product
+# Extending/Adopting the Theia IDE
+
+This guide provides an overview on how to extend and customize the Theia IDE to your own custom IDE or tool. In this scenario, the Eclipse Theia IDE is an example product used as a reference on how to build desktop IDE-like products based on the Eclipse Theia framework. If you just want to use the Theia IDE, see the [user guide](/docs/user_getting_started)
+
+Please note that adopting the Theia IDE as a basis is just one of several ways to get started with building a Theia-based application. We recommend reading the article "[Build your own IDE/Tool](/docs/composing_applications)" as a first step. Furthermore, this guide is focused on building a desktop app. We also provide an [experimental Docker version](https://github.com/eclipse-theia/theia-blueprint?tab=readme-ov-file#docker-build) of the Theia IDE as an alternative.
 
-This guide provides an overview on how to extend and customize the Theia IDE to your own custom IDE or tool. In this scenario, the Eclipse Theia IDE is an example product used as a reference on how to build desktop IDE-like products based on the Eclipse Theia framework.
 The Theia IDE assembles a selected subset of existing Theia features and extensions.
 We provide installers for the Theia IDE to be downloaded (see links below).
-In the respective git repository you can also find the source code for the Theia IDE and its installers.
+In the [respective git repository](https://github.com/eclipse-theia/theia-blueprint) you can also find the [source code for the Theia IDE and its installers](https://github.com/eclipse-theia/theia-blueprint).
```

Comment:
Both links point to https://github.com/eclipse-theia/theia-blueprint , so I think we could also make one of them a regular text. But this is not a blocker from my pov, so if you prefer it this way, you may also merge immediately.